app/degreePlanner/detailPlanner.css

```diff
@@ -1,166 +1,181 @@
 .degreePlanContainer {
-    display: flex;
-    flex-direction: row;
-    width: 100%;
-    align-items: space-between;
-
+  display: flex;
```

Comment:
Please don't edit any css that you didn't touch!

Reply:
it indentation issue, nothing else !